### PR TITLE
Added few changes after testing

### PR DESCRIPTION
**Pull request by ayyoob**

### CODE REVIEW

**Comment by sumedharubasinghe** [comment]
Merging #128.

File: modules/iot-extensions/components/das-extensions/org.wso2.carbon.event.input.adapter.extensions/src/main/java/org/wso2/carbon/event/input/adapter/extensions/util/PropertyUtils.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                                                                                                "localhost"));
+        return urlWithPlaceholders;
+    }
+
+    private static int getPortOffset() {
+        ServerConfiguration carbonConfig = ServerConfiguration.getInstance();
+        String portOffset = System.getProperty("portOffset",
+                                               carbonConfig.getFirstProperty(CARBON_CONFIG_PORT_OFFSET));
+        try {
+            if ((portOffset != null)) {
```
Comment (prabathabey) [inline modules/iot-extensions/components/das-extensions/org.wso2.carbon.event.input.adapter.extensions/src/main/java/org/wso2/carbon/event/input/adapter/extensions/util/PropertyUtils.java:46]: Remove redundant pair of brackets used in the if clause.

File: modules/iot-extensions/components/das-extensions/org.wso2.carbon.event.input.adapter.extensions/src/main/java/org/wso2/carbon/event/input/adapter/extensions/util/PropertyUtils.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        urlWithPlaceholders = Utils.replaceSystemProperty(urlWithPlaceholders);
+        urlWithPlaceholders = urlWithPlaceholders.replaceAll(MQTT_PORT, "" + (DEFAULT_MQTT_PORT + getPortOffset()));
+        urlWithPlaceholders = urlWithPlaceholders.replaceAll(MQTT_BROKER_HOST, System.getProperty(DEFAULT_CARBON_SERVER_HOST_PROPERTY,
+                                                                                                "localhost"));
+        return urlWithPlaceholders;
+    }
+
+    private static int getPortOffset() {
+        ServerConfiguration carbonConfig = ServerConfiguration.getInstance();
+        String portOffset = System.getProperty("portOffset",
```
Comment (prabathabey) [inline modules/iot-extensions/components/das-extensions/org.wso2.carbon.event.input.adapter.extensions/src/main/java/org/wso2/carbon/event/input/adapter/extensions/util/PropertyUtils.java:43]: We might probably have some constants/util methods defined in carbon utils upon well known attributes such as "portOffset". Use those as much as possible to make the code more maintainable.

File: modules/iot-extensions/components/das-extensions/org.wso2.carbon.event.input.adapter.extensions/src/main/java/org/wso2/carbon/event/input/adapter/extensions/util/PropertyUtils.java (hunk 1 above)
Comment (ayyoob) [inline modules/iot-extensions/components/das-extensions/org.wso2.carbon.event.input.adapter.extensions/src/main/java/org/wso2/carbon/event/input/adapter/extensions/util/PropertyUtils.java:46]: Fixed with https://github.com/wso2/carbon-device-mgt-plugins/pull/309/commits/4ffa47f0b4514143e4ec32f2bf59179f70a03be5

File: modules/iot-extensions/components/das-extensions/org.wso2.carbon.event.input.adapter.extensions/src/main/java/org/wso2/carbon/event/input/adapter/extensions/util/PropertyUtils.java (hunk 2 above)
Comment (ayyoob) [inline modules/iot-extensions/components/das-extensions/org.wso2.carbon.event.input.adapter.extensions/src/main/java/org/wso2/carbon/event/input/adapter/extensions/util/PropertyUtils.java:43]: Yes that should be the correct way but there aren't any constants variable found in carbon utils. Even in carbon utils port offset is retrieved without a constant. https://github.com/wso2/carbon-kernel/blob/4.4.x/core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/CarbonUtils.java#L882-L886

**Pull request merged**